site/content/posts/2023-08-08-v0.25.0-impersonation-proxy-with-external-certs.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Pinniped v0.25.0: With External Certificate Management for the Impersonation Proxy and more"
+slug: v0-25-0-external-cert-mgmt-impersonation-proxy
+date: 2023-08-08
+author: Joshua T. Casey
+image: https://images.unsplash.com/photo-1618075254460-429d47b887c7?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2148&q=80
+excerpt: "With v0.25.0 you get external certificate management for the impersonation proxy, easier scheduling of the kube-cert-agent, and more"
+tags: ['Joshua T. Casey','Ryan Richard', 'Benjamin Petersen', 'release']
+---
+
+![Friendly seal](https://images.unsplash.com/photo-1618075254460-429d47b887c7?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2148&q=80)
+*Photo by [karlheinz_eckhardt Eckhardt](https://unsplash.com/@karlheinz_eckhardt) on [Unsplash](https://unsplash.com/s/photos/seal)*
+
+With Pinniped v0.25.0 you get the ability to configure an externally-generated certificate for the Impersonation Proxy to serve TLS.
```

Comment:
Let's expand on this sentence into a paragraph.  I see the link below to "more about the impersonation proxy" but I bet for SEO purposes we could do well to summarize something here.  My comment up by the title about something like "providing a Solution for Kubernetes Authentication with an OIDC Identity Provider with your own certificates or PKI" as well as a sentence about how the Impersonation Proxy provides Pinniped the ability to authenticate to public clouds.